rvgo/test/evm_test.go

```diff
@@ -54,7 +54,7 @@ func fakeHeader(n uint64, parentHash common.Hash) *types.Header {
 	return &header
 }
 
-func loadRISCVContractCode(t *testing.T) *Contract {
+func loadRISCVContractCode(t require.TestingT) *Contract {
```

Comment:
May I ask why do we need this diff?

Reply:
for supporting fuzz test. `require.TestingT` is an interface which supports `testing.T` and `testing.F`.